signer/src/config/mod.rs

```diff
@@ -253,6 +255,18 @@ pub struct SignerConfig {
     /// arrives. The default here is controlled by the
     /// [`MAX_DEPOSITS_PER_BITCOIN_TX`] constant
     pub max_deposits_per_bitcoin_tx: NonZeroU16,
+
+    /// Configures a DKG re-run Bitcoin block height. If this is set and DKG
+    /// has already been run, the coordinator will attempt to re-run DKG after
+    /// this block height at most once. If DKG has never been run, this
+    /// configuration has no effect.
+    pub dkg_rerun_bitcoin_height: Option<NonZeroU64>,
```

Comment:
nit: `at most once` seems to contradict `dkg_rounds_target` (_the coordinator will continue to run DKG rounds until this number of rounds is reached_)

Reply:
Yeah good catch, missed to update this after adding the other param

Reply:
[200ca30](https://github.com/stacks-network/sbtc/pull/1204/commits/200ca30ba7e6f808dbaf987571e3fdec5b46c1e7)